src/sp_repo_review/checks/general.py

```diff
@@ -50,7 +50,12 @@ class PY003(General):
     @staticmethod
     def check(package: Traversable) -> bool:
         "Projects must have a license"
-        return len([p for p in package.iterdir() if "LICENSE" in p.name]) > 0
+        spellings = ("LICENSE", "LICENCE", "COPYING")
```

Comment:
I was thinking this should be a set, but for three items, the list is actually ~12% faster. I assume sets take over around 4 elements or so. FYI, `startswith` is actually 2x slower for somewhat short strings. `in` seems to be special, probably because it avoids a function call.

(Speed doesn't actually matter here at all, but it was fun to look at).

Reply:
The real benifit of a set is the fast random lookup, due to the underlying hashmap. But as we are going through this sequentially I wouldn't have thought it would ever be faster.

Reply:
Ahh, yes, `in` is not applied to the set. :facepalm: